tests/foreman/ui/test_activationkey.py

```diff
@@ -379,15 +371,17 @@ def test_negative_create_activation_key_2(self):
         @BZ: 1177158
 
         """
-
-        name = gen_string("alpha", 10)
-        description = gen_string("alpha", 1001)
+        name = gen_string('alpha', 10)
+        description = gen_string('alpha', 1001)
         with Session(self.browser) as session:
-            make_activationkey(session, org=self.org_name,
-                               name=name, env=ENVIRONMENT,
-                               description=description)
-            self.assertIsNotNone(self.activationkey.wait_until_element
-                                 (common_locators["common_haserror"]))
+            make_activationkey(
+                session, org=self.org_name, name=name, env=ENVIRONMENT,
+                description=description)
+            self.assertIsNotNone(
+                self.activationkey.wait_until_element(
+                    common_locators['common_haserror']
+                )
```

Comment:
Actually both formats are valid, but I think when having many arguments is better to place every element on its own line. For this module this applies for all make_activationkey calls.

On the other hand, the wait_until_element inside the assert I think is better to place the closing ) on it's own line to improve readability. But when not inside another call, I think the following format is ok:

```python
self.activationkey.wait_until_element(
    common_locators['common_haserror'])
```